recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,47 @@
+
+{% set name = "unifunc" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9eaa196505dabe471b3ac790f3e84770919f53728de2f6f0ecbbb550812fde97
+  path: ..
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - pip
+    - python
+    - setuptools
+  run:
+  - nltk
+  - numpy
+  - python
+  - requests
```

```suggestion
  run:
    - nltk
    - numpy
    - python
    - requests
```
You don't have any requirements listed in your _setup.py_ or _pyprpoject.tom_.
Are these all? Might be good to add them to your project as well.